assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/convert.py

```diff
@@ -36,6 +36,7 @@ def _prepare_artifacts_dict(input_dir: Path) -> Dict:
     artifacts_dict = {
         MMDetLiterals.CONFIG_PATH: os.path.join(input_dir, metadata.get("pytorch_model_path")),
         MMDetLiterals.WEIGHTS_PATH: os.path.join(input_dir, metadata.get("model_weights_path_or_url")),
+        MMDetLiterals.METAFILE_PATH: os.path.join(input_dir, metadata.get("model_metafile_path")),
```

Comment:
should we also add model_defaults.json??

Reply:
+1

Reply:
This model_ddefaults.json file is not generated in model selector component. 
This mlflow conversion component can only copy files from output of model selector component to the MLflow folder. Hence, this can't be done in this component, we have to add it manually later.